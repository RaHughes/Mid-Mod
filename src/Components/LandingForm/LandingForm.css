.landing_form {
  display: flex;
  justify-content: space-between;
}

button {
  background-color: teal;
  color: white;
  height: 30px;
}