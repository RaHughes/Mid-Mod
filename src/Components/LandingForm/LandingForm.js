import React, { Component } from 'react';

class LandingForm extends Component {
  constructor() {
    super()
    this.state = {
      name: '',
      date: '',
      time: '',
      guests: ''
    }
  }
}

export default LandingForm