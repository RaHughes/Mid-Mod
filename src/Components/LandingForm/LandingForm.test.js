import React from 'react';
import { shallow } from 'enzyme';
import LandingForm from './LandingForm';

describe('LandingForm', () => {
  it('Should match snapshot', () => {
    const wrapper = shallow(<LandingForm />);
    expect(wrapper).toMatchSnapshot();
  })

  it('should update shate when handleChange is called', () => {
    const wrapper = shallow(<LandingForm />)
    const event = { target: { name: 'name', value: 'Leonardo Dicaprio' } };
    const outcome = 'Leonardo Dicaprio';
    wrapper.instance().handleChange(event);
    expect(wrapper.state('name')).toEqual(outcome);
  })
})