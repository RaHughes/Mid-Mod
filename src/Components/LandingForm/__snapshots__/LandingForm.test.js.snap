// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingForm Should match snapshot 1`] = `
<div>
  <input
    name="name"
    onChange={[Function]}
    placeholder="Name"
    type="text"
    value=""
  />
  <input
    name="date"
    onChange={[Function]}
    placeholder="Date (MM/DD)"
    type="text"
    value=""
  />
  <input
    name="time"
    onChange={[Function]}
    placeholder="Time"
    type="text"
    value=""
  />
  <input
    name="number"
    onChange={[Function]}
    placeholder="Number of Guests"
    type="text"
  />
  <button
    onClick={[Function]}
  >
    Make Reservation
  </button>
</div>
`;
